exasol/ds/sandbox/runtime/ansible/roles/jupyter/files/notebook/work_in_progress/text_ai/txaie_init.ipynb

```diff
@@ -11,6 +11,12 @@
     "\n",
```

Comment:
<p>Suggested changes:</p><p> </p><p>first sentence: "To be able to store and run the models used in this demo, your database should meet the following requirements"</p><p> </p><p>"wipe the content" =&gt; "wipe out the content"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/exasol/ai-lab/pull/364/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>